src/test/packages/nginx/refs/zarf.yaml

```diff
@@ -0,0 +1,28 @@
+kind: ZarfPackageConfig
+metadata:
+  name: nginx
+  version: 0.0.2
+
+components:
+  - name: nginx-remote
+    required: true
+    manifests:
+      - name: simple-nginx-deployment
+        namespace: nginx
+        files:
+          - deployment.yaml
+    actions:
+      onDeploy:
+      # the following checks were computed by viewing the success state of the package deployment
+      # and creating `wait` actions that match
+        after:
+          - wait:
+              cluster:
+                kind: deployment
+                name: nginx-deployment
+                namespace: nginx
+                condition: available
+    # image discovery is supported in all manifests and charts using:
+    # zarf prepare find-images
+    images:
+      - docker.io/library/nginx:1.26.0
```

Comment:
Nice! Good call making the image be a different version, if that's not called out in the `metadata.description` I'd call it out here